stable/artifactory/README.md

```diff
@@ -1134,35 +1134,11 @@ helm upgrade --install nginx-ingress --namespace nginx-ingress center/kubernetes
 ```
 This will start sending your Artifactory logs to the log aggregator of your choice, based on your configuration in the `filebeatYml`
 
-### Prometheus Metrics
+### Log Analytics
 
-If you want to enable Prometheus metrics you can use the `metrics` configuration options. By default this option requires that the Promtheus Operator already be deployed and the associated CRDs created.
+#### FluentD, Prometheus and Grafana
 
-The simplest way is to install Artifactory with the following command:
-
-```bash
-helm upgrade --install artifactory --namespace artifactory --set metrics.enabled=true center/jfrog/artifactory
-```
-
-This will create a new service exposing the Prometheus metrics as well as a ServiceMonitor object for the Prometheus Operator to start scraping.
-
-The `ServiceMonitor` creation can be disabled by setting `metrics.serviceMonitor.enabled` to `false` for environments that are not using the Prometheus Operator.
-
-```bash
-helm upgrade --install artifactory --namespace artifactory --set metrics.enabled=true --set metrics.serviceMonitor.enabled=false center/jfrog/artifactory
-```
-
-NOTE: Enabling this does NOT create a container which actually parses the log files for metrics.  See the Fluentd section below.
-
-### Fluentd
-
-The suggested way to parse the log files for metrics is to install Fluentd as a sidecar container.  This can be done with the `fluentd` configuration options.
-
-```bash
-helm upgrade --install artifactory --namespace artifactory --set fluentd.enabled=true center/jfrog/artifactory
-```
-
-This will install Artifactory with Fluentd running as a sidecar container sharing the persistent volume where the log files are written.
+To configure Prometheus and Grafana to gather metrics from Artifactory through the use of FluentD, please refer - https://github.com/jfrog/log-analytics-prometheus
```

Comment:
Can we update this to below:

To configure Prometheus and Grafana to gather metrics from Artifactory through the use of FluentD, please refer to the log analytics repo:

https://github.com/jfrog/log-analytics-prometheus

That repo contains a file `artifactory-values.yaml` that can be used to deploy Prometheus, Service Monitor, and Grafana with this chart.